browsers/chrome.json

```
@@ -574,16 +574,23 @@
         "83": {
           "release_date": "2020-05-19",
           "release_notes": "https://chromereleases.googleblog.com/2020/05/stable-channel-update-for-desktop_19.html",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "83"
         },
         "84": {
-          "status": "beta",
+          "release_date": "2020-07-14",
+          "release_notes": "https://chromereleases.googleblog.com/2020/07/stable-channel-update-for-desktop.html",
+          "status": "current",
           "engine": "Blink",
           "engine_version": "84"
         },
         "85": {
+          "status": "beta",
+          "engine": "Blink",
+          "engine_version": "85"
+        },
+        "86": {
           "status": "nightly",
           "engine": "Blink",
           "engine_version": "85"
```

Unless something very unusual has happened, I'd expect this version to have changed as well:

```suggestion
          "engine_version": "86"
```